src/core/ext/filters/client_channel/dynamic_filters.cc

```diff
@@ -163,10 +163,16 @@ RefCountedPtr<DynamicFilters> DynamicFilters::Create(
     // Channel stack creation failed with requested filters.
     // Create with lame filter instead.
     grpc_error_handle error = p.second;
-    grpc_arg error_arg = MakeLameClientErrorArg(error);
+    grpc_error_handle* error_ptr = nullptr;
+    if (error != GRPC_ERROR_NONE) {
+      error_ptr = new grpc_error_handle();
```

Comment:
I don't think we need any dynamic allocation here.  I think we can just do:

```
grpc_arg error_arg = MakeLameClientErrorArg(&error);
```

Reply:
Good catch! Done.